cegs_portal/search/views/v1/dna_features.py

```diff
@@ -71,6 +71,7 @@ def get_sig_only(value):
 
     def get(self, request, options, data, id_type, feature_id):
         feature_reos = []
+        reo_page = []
```

Comment:
I added it because of this error. Want it in a different location?
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/a6f19cb5-ff85-4a0d-be0f-e52457bb75f2)

Reply:
Ah ok, I see what's going on now. I think it should be None because of line 116 `if any(f[3] is not None for f in feature_reos):`, right?

Reply:
Got it